website/pages/blog/announcing-graphql-network-inspector.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Announcing GraphQL Network Inspector
+tags: [graphql, graphql-network-inspector]
+authors: [warrenjday]
+date: 2023-02-10
+description: A better way to debug network traffic with GraphQL
+image: /blog-assets/announcing-graphql-network-inspector/cover.png
+thumbnail: /blog-assets/announcing-graphql-network-inspector/thumbnail.png
+---
+
+GraphQL Network Inspector is a dedicated devtools extension which allows you to inspect and debug
+GraphQL queries and mutations from network traffic.
+
+The extension has seen great adoption since its initial release and now I'm excited to announce it's
```

Comment:
do we want to think about the fact we have both GraphQL Inspector and GraphQL Network Inspector?
Would it be confusing to people?
I'm good with both options or with renaming, up to you

Reply:
I could see why this would be confusing, the extension once installed is called "GraphQL Network"

<img width="546" alt="Screenshot 2023-02-16 at 19 20 29" src="https://user-images.githubusercontent.com/2871038/219466060-86958d95-25aa-46f2-b703-eec7618fa9a5.png">

So we can go with that, or refer to it here as "GraphQL Network Devtools"

Reply:
Happy with either one.

Reply:
interesting idea!
I love both "GraphQL Network" and "GraphQL Network Devtools"
I wonder what others are thinking as well

Reply:
ok we all agreed that your current name, "GraphQL Network Inspector", is best, so change nothing!